client/src/pages/chatlink/index.js

```diff
@@ -29,17 +34,29 @@ const App = () => {
     renderGoogleReCaptcha(captchaIns, setCaptchToken, 'captcha');
   };
 
+  const toggleTheme = () => {
+    setDarkMode(!darkMode);
+  };
+
   useEffect(() => {
     initCaptcha();
   }, []);
 
   return (
     <>
-      <div className="app link-generation--page">
-        <div className="header">Because privacy matters!</div>
-        <div className="section--default">
-          <div className="title">Generate temporary link and start chatting without worrying.</div>
-          <div className="description">
+      <div className={stylesLocal.linkGenerationPage}>
+        <div
+          className={`${stylesLocal.header} 
+          ${darkMode === true ? stylesLocal.darkModeHeader : stylesLocal.lightModeHeader}`}
+        >
+          Because privacy matters!
+          {darkMode === true ? <FiSun onClick={toggleTheme} /> : <FiMoon onClick={toggleTheme} />}
```

Comment:
Create a separate component for the theme toggling button (including the context setter).
Re-use the button on both pages i.e. link generation and messaging. 
(Mentioned previously)

Reply:
Done